katcp/inspecting_client.py

```diff
@@ -109,12 +174,15 @@ class InspectingClientAsync(object):
     """
 
     sync_timeout = 5
```

Comment:
I also tested this on dev32 and it is working well. 

However, my suggestion is it increase the sync_timeout to 10s. It is a timeout after all and if it completes before the timeout triggers then all is OK. If it actually still was busy trying to complete the syncing, then backing off and retrying is just adding to the load. 
The sensor-list timeouts reported were all between 1.5 to 3.8 seconds. I tested on dev32 with 8s timeout and it all completed successfully on first time sync then. I suggest to set the sync_timeout to 10s though. That way we have a good chance of syncing up on the first try, and does not hit the lower level device too hard by resyncing often and we still have the retry mechanism if that does not work.